forgot_password/handlers/forgot_password.py

```diff
@@ -124,7 +125,9 @@ def test_forgot_password_email(email,
                                    html_template=None,
                                    subject=None,
                                    sender=None,
-                                   reply_to=None):
+                                   reply_to=None,
+                                   sender_name='',
```

Comment:
Does the underlying library works if you use `None`, as far as I checked, the underlying library is smart enough to omit the name if the value is false.

Well... just trying to make this consistent with other params. : )